modules/serverless-rn-1-4-0.adoc

```diff
@@ -0,0 +1,33 @@
+// Module included in the following assemblies:
+//
+// * serverless/release-notes.adoc
+
+[id="serverless-rn-1-4-0_{context}"]
+= Release Notes for Red Hat {ServerlessProductName} Technology Preview 1.4.0
+
+[id="new-features-1-4-0_{context}"]
+== New features
+* {ServerlessProductName} 1.4.0 is available on {product-title} 4.2 and newer versions.
+* {ServerlessProductName} has been updated to use Knative Serving 0.11.1.
+* {ServerlessProductName} has been updated to use Knative Client (`kn` CLI) 0.11.0.
+* {ServerlessProductName} has been updated to use Knative Serving Operator 0.11.1.
+* Knative Client is now available for download through the *Command Line Tools* page in the {product-title} web console.
+* The `KnativeServing` object's API group has changed in this release from `serving.knative.dev` to `operator.knative.dev`. You will need to adjust any of your scripts or applications that rely on the old API group to use the new API group. The {ServerlessProductName} installation instructions have been updated to use the new API group.
++
+When upgrading from {ServerlessProductName} 1.3.0 to 1.4.0, the {ServerlessOperatorName} will create a `KnativeServing` custom resource (CR) in the new API group for you. This CR will be a mirror of the `KnativeServing` CR in the old group that was used in {ServerlessProductName} 1.3.0.
++
+If you need to keep using the old group temporarily, you can use the old CR as before. However, this CR is deprecated and will eventually be removed.
++
+Once you have updated references to the new API group, you can remove any older CR versions and use the newly deployed `KnativeServing` CR instead.
++
+[IMPORTANT]
+====
+If a previous version of the CR exists, changes to the new CR will be overwritten by the {ServerlessOperatorName}. While the old CR is still active, all changes need to be made to that CR.
+====
+
+[id="fixed-issues-1-4-0_{context}"]
+== Fixed issues
+* Connecting to a private, cluster local Knative Service from a namespace that was not part of the `knative-serving-ingress` Service Mesh was failing on `i/o timeout`. This issue is now fixed.
+
+[id="known-issues-1-4-0_{context}"]
+== Known issues
```

Comment:
Let's add

- Unqualified usages of `knativeserving` in `oc` commands (like `oc get knativeserving -n knative-serving`) may no longer work because of the aforementioned migration to a new API group. Use the explicit fully-qualified format (like `oc get knativeserving.operator.knative.dev -n knative-serving`) instead.

https://issues.redhat.com/browse/SRVKS-414 for reference.